src/components/Slider/Slider.css

```diff
@@ -61,7 +65,7 @@
 }
 
 .Slider--disabled {
-  opacity: 0.4;
+  opacity: var(--vkui--opacity_disable);
```

Comment:
```suggestion
  opacity: var(--vkui--opacity_disable_accessibility);
```

@Fliqle говорил, что надо везде прозрачность в 64% использовать

<img width="467" alt="image" src="https://user-images.githubusercontent.com/5850354/173095939-60177656-e235-4be5-bcfd-8c44c8b16ab8.png">

> фото из ЛС